core/cautils/customerloader.go

```diff
@@ -329,7 +329,7 @@ func (c *ClusterConfig) updateConfigEmptyFieldsFromCredentialsSecret() error {
 		}
 	}
 
-	if jsonConf, ok := secrets.Items[0].Data["accessKey"]; ok {
+	if jsonConf, ok := secrets.Items[0].Data["access-key"]; ok {
```

Comment:
You can't change that..

https://github.com/kubescape/helm-charts/blob/main/charts/kubescape-operator/templates/configs/cloud-secret.yaml#L13C3-L13C12